### PR TITLE
Delivery/feature/app structure

### PR DESCRIPTION
**Pull request by Youssef-Ramadan-Y**
- add packages structure
- add dependencies 
- add di
- add bases in presentation
- and main screen

### CODE REVIEW

File: client_delivery/androidApp/src/androidMain/AndroidManifest.xml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <application
+        android:name="com.beepbeep.App"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
+        android:usesCleartextTraffic="true"
```
Comment (AzizaHelmy) [inline client_delivery/androidApp/src/androidMain/AndroidManifest.xml:11]: This line isn't required

File: client_delivery/shared/build.gradle.kts [hunk 2]
```diff
@@ -26,21 +32,45 @@ kotlin {
...
 
     sourceSets {
-
         val commonMain by getting {
             dependencies {
                 implementation(libs.compose.runtime)
                 implementation(libs.compose.foundation)
                 implementation(libs.compose.material3)
+                api(libs.compose.image.loader)
+
```
Comment (AzizaHelmy) [inline client_delivery/shared/build.gradle.kts:41]: Why we need image loader in our app

**Review by AzizaHelmy — APPROVED** [review]

File: client_delivery/shared/build.gradle.kts (hunk 2 above)
Comment (Youssef-Ramadan-Y) [reply client_delivery/shared/build.gradle.kts:41]: ok removed

File: client_delivery/androidApp/src/androidMain/AndroidManifest.xml (hunk 1 above)
Comment (Youssef-Ramadan-Y) [reply client_delivery/androidApp/src/androidMain/AndroidManifest.xml:11]: ok I got it we don't need it any more know thx

**Review by Hassan3Ayman — APPROVED** [review]

**Pull request merged**